enginetest/enginetests.go

```diff
@@ -1700,6 +1701,507 @@ func TestTracing(t *testing.T, harness Harness) {
 	require.Equal(expectedSpans, spanOperations)
 }
 
+func TestColumnDefaults(t *testing.T, harness Harness) {
```

Comment:
Great tests. Only comment would be to replace the comments in this function with calls to t.Run()

Reply:
I should have done this the first time 🤦